### ISSUE TITLE
Contained Databases Error when using BulkSaveChange and BulkInsert  

### ISSUE THREAD

**Issue by NperronNmedia** [zzzprojects/EntityFramework-Extensions#361]
### Description
Hi, i was using the nugget Z.EntityFramework.Extensions to make BulkInsert and BulkSaveChange on a contained database. Last week i have update the nugget from version 4.0.86 to 4.0.93 and for some reason i now have an error when i try to make a BulkSaveChange or a BulkInsert. Everything was good before and i'm using the nugget for almost one year now. 

I try to check if there is any changelog to see the change between 4.0.86 and 4.0.93 , but i found nothing.

### Exception

```
Exception message: User-named PRIMARY KEY constraint 'PK_#ZZZProjects_a09094dc_86db_41cf_a35e_0265891fcdf3' is not allowed on temp table '#ZZZProjects_a09094dc_86db_41cf_a35e_0265891fcdf3' because it is being created in a contained database. Please consult the Books Online topic Understanding Contained Databases for more information on contained databases.
Could not create constraint or index. See previous errors.

```

### Further technical details
- EF version: 6.2.0
- EF Extensions version: 4.0.93
- Database Provider: SQL Server

**Comment by JonathanMagnan** [comment]
Hello @NperronNmedia ,

The issue probably started with the v4.0.88: https://github.com/zzzprojects/EntityFramework-Extensions/releases/tag/3.0.88 (See: The temporary table now always create a clustered index in SQL Server for the ZZZ_Index column (was already the case for SQL Azure))

We will look this week how we can handle this feature or `contained database` automatically.

Best Regards,

Jon

**Comment by JonathanMagnan** [comment]
Hello @NperronNmedia ,

The v4.0.95 has been released.

A `clustered index` is now created for performance optimization instead of creating a primary key.

Let us know if everything is now working as expected in this new version.

Best Regards,

Jon

**Comment by JonathanMagnan** [comment]
Hello @NperronNmedia,

Since our last conversation did you get the time to try the new version v4.0.95. 

Let me know if further assistance is necessary.

Best regards, 

Jon

**Comment by JonathanMagnan** [comment]
Hello @NperronNmedia,

A simple follow-up to see if your issue was resolved the v4.0.95.

Don't hesitate to contact me if you have any questions. 

Best regards, 

Jon
